txclib/commands.py

```diff
@@ -85,12 +85,19 @@ def cmd_init(argv, path_to_tx):
 
     if not options.skipsetup and not options.no_interactive:
         logger.info(messages.running_tx_set)
-        cmd_set([], path_to_tx)
+        cmd_config([], path_to_tx)
     else:
         logger.info("Done.")
 
 
 def cmd_set(argv, path_to_tx):
+    """Add local or remote files under transifex. Warning: \
+This command will be deprecated in a future release of the \
```

Comment:
The doc string is used by the `tx help` command and I was trying to avoid this: 
```
Transifex command line client.

Available commands are:
  config         	Add local or remote files under transifex
  delete         	Delete an accessible resource or translation in a remote server.
  help           	List all available commands
  init           	Initialize a new transifex project.
  pull           	Pull files from remote server to local repository
  push           	Push local files to remote server
  set            	Add local or remote files under transifex. Warning:
       This command will be deprecated in a future release of the
       client. You should use the `tx config` command.

  status         	Print status of current project
```

Reply:
I see, ok then!